### ISSUE TITLE
Add a Custom Advancement Datapack

### ISSUE THREAD

**Issue by connor7000** [mc-ascendancy/ascendancy#9]
From **boppolopian#7552**

"Custom achievements that can act as a tutorial similar to sevtech."

**Comment by connor7000** [comment]
I assume this player is referring to [SevTech: Ages](https://sevtechages.gamepedia.com/SevTech:_Ages_Wiki#:~:text=SevTech%3A%20Ages%20is%20a%20massive,progression%20experience%20with%20a%20purpose.) which I know little about.

@NotARobotII, as we discussed a few weeks back if we decide to ever finish up that custom advancement datapack, we should definitely integrate AscendNations somehow. Perhaps, grant some advancements for reaching some level of Passive Power / Nation Power. Or for claiming _Rifts_ or maybe for successfully resupplying an Outpost?

Not sure how practical this implementation might be, and perhaps @VolticFroogo can address that!

**Comment by VolticFroogo** [comment]
> Not sure how practical this implementation might be, and perhaps @VolticFroogo can address that!

Yeah, this would probably be a fairly simple addition to AscendNations as datapack advancements seem to work very well using the modern Spigot API.
